### PR TITLE
2023 05 release notes

### PR DESCRIPTION
**Pull request by prestwick**
- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/install-systemlink-enterprise/blob/master/CONTRIBUTING.md).

### What does this Pull Request accomplish?

Release notes for 2023-05

### Why should this Pull Request be merged?

Standard release process 

### What testing has been done?

We will need to verify the list of provided containers is accurate before merging the PR

### CODE REVIEW

**Review by JD-Robertson — APPROVED** [review]

File: release-notes/2023-05/README.md [hunk 1]
```diff
@@ -0,0 +1,179 @@
...
+
+### 3rd Party Containers
+
+argoproj/argocli:v3.3.8-linux-amd64
+
+argoproj/argoexec:v3.3.8-linux-amd64
+
+argoproj/workflow-controller:v3.3.8-linux-amd64
+
+bats/bats:1.9.0
```
Comment (pvallone) [inline release-notes/2023-05/README.md:139]: We can remove this too. We disabled the feature in Grafana that used this image.

File: release-notes/2023-05/README.md [hunk 2]
```diff
@@ -0,0 +1,179 @@
...
+
+bitnami/minio:2023.4.28-debian-11-r0
+
+strimzi/jmxtrans:0.34.0
```
Comment (pvallone) [inline release-notes/2023-05/README.md:143]: We can omit the following:
- `strimzi/jmxtrans:0.34.0`
- `strimzi/kafka-bridge:0.25.0`
- `strimzi/kafka:0.34.0-kafka-3.3.1`
- `strimzi/kafka:0.34.0-kafka-3.3.2`
- `strimzi/kaniko-executor:0.34.0`
- `strimzi/maven-builder:0.34.0`

We removed a few of the Strimzi images in [this PR](https://dev.azure.com/ni/DevCentral/_git/Skyline/pullrequest/474769?_a=files&path=/Enterprise/SystemLinkAdmin/helm/Chart.yaml).

**Review by pvallone — APPROVED** [review]

File: release-notes/2023-05/README.md (hunk 1 above)
Comment (prestwick) [reply release-notes/2023-05/README.md]: done

Comment (pvallone) [reply release-notes/2023-05/README.md]: Sorry, I didn't highlight the correct line. I meant to suggest we remove `bats/bats:1.9.0`. We should keep `argoproj/workflow-controller:v3.3.8-linux-amd64`.

Comment (prestwick) [reply release-notes/2023-05/README.md]: ok, I figured that was the case given earlier discussions about the bats container. I have removed that line and left the one for argo workflow-controller.

Comment (g-m-o-n-e-y) [reply release-notes/2023-05/README.md]: `bats` was promoted when we built the release ([link](https://dev.azure.com/ni/DevCentral/_build/results?buildId=4828580&view=logs&j=971690b9-2a65-5227-cd0f-d4c8dd6cddb7&t=d719c0b4-78e9-5373-5290-0e4ae67a9791&l=18)). Just want to double-check with @pvallone that this is OK and (I assume) due to being deployed before.

**Review by g-m-o-n-e-y — APPROVED** [review]

File: release-notes/2023-05/README.md (hunk 1 above)
Comment (pvallone) [reply release-notes/2023-05/README.md]: Yes, that's fine. It looks like we disabled the feature that pulls in the image shortly after the release was built, so `bats/bats:1.9.0` is technically in the 2023-05 release.

Comment (prestwick) [reply release-notes/2023-05/README.md]: I've re-added bats

**Pull request merged**